modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,121 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage"]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+Before you modify the machine set, you must manually generate the `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Configured your Azure portal account.
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
```

Comment:
@aleskandro Hi Alessandro! PTAL at this procedure when you get the chance, thank you!